TECHNICAL_BOARD.md

```diff
@@ -0,0 +1,53 @@
+# **Technical Board**
+
+The ERDDAP™ Technical Board is responsible for the technical direction that ERDDAP™ takes. The ERDDAP™ Technical Board is responsible for the technical direction of ERDDAP™. It makes decisions on prioritization of work, feature roadmaps, dependencies, any processes that impact development work, and any other matter which requires technical supervision in ERDDAP™. The board will also review and approve design documents for significant new features.Technical board terms are 2 years with the option of extending. Technical board members are also expected to be contributing to the project, in particular through pull requests and code reviews.
+
+## **Purpose**
+
+The Technical Board is responsible for the following documents and processes:
+
+* **Review and Approve new features** that are proposed in GitHub Discussions.
+
+* **ERDDAP™ Priority Projects** list, which determine projects that we are working on. Check out the current [priority projects list](PRIORITY_PROJECTS.md).
+
+* **ERDDAP™ Release Feature Goals**, which determine specific features which we aim to include in each release of ERDDAP™. These are documented in the GitHub project for each release.
+
+* **ERDDAP™ Style Guide** which specifies coding styles and practices to be followed in ERDDAP™ code.The ERDDAP™ Technical Board operates on the following basis:* Where possible communication of the Technical Board should be public and asynchronous. In many cases this means utilizing GitHub Discussions and Issues.
```

Comment:
Should the sentence starting with 'The ERDDAP Technical Board operates...' be moved to a separate line or bullet?  Seems unrelated to the Style Guide.

Reply:
Good catch! Done.